### PR TITLE
Updated 600,000 to 700,000 active installs [MAILPOET-5541]

### PR DESCRIPTION
**Pull request by NeosinneR**
## Description

_N/A_

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5541]

## After-merge notes

_N/A_

## Tasks

- [x] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [x] 🚫 I added sufficient test coverage
- [x] 🚫 I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-5541]: https://mailpoet.atlassian.net/browse/MAILPOET-5541?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/js/src/landingpage/content.tsx [hunk 1]
```diff
@@ -29,7 +29,10 @@ function Content() {
         <br />
         <br />
         <Heading level={4}>
-          {__('Powering email marketing for 600,000+ websites', 'mailpoet')}
+          {__(
+            'Powering email marketing for [count] websites',
+            'mailpoet',
+          ).replace('[count]', '700,000+')}
```
Comment (websupporter) [inline mailpoet/assets/js/src/landingpage/content.tsx]: We usually use `sprintf()`, I would also suggest a `// translators` comment before explaining the `%s` and you would need to import `sprintf()`
```suggestion
          {sprintf(__(
            'Powering email marketing for %s websites',
            'mailpoet',
          ), '700,000+')}
```

**Review by websupporter — CHANGES_REQUESTED** [review]
see comment above

**Review by websupporter — DISMISSED** [review]
lgtm. thank you @NeosinneR

**Comment by NeosinneR** [comment]
@websupporter I had to force push due to prettier failing

**Review by websupporter — APPROVED** [review]
lgtm.

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**